extensions/integration-tests/src/notebook.test.ts

```diff
@@ -380,6 +389,18 @@ class NotebookTester {
 		assert(clearedOutputs, 'Outputs of all the code cells from Python notebook should be cleared');
 		console.log('After clearing cell outputs');
 	}
+
+	async verifyClearOutputs(notebook: azdata.nb.NotebookEditor): Promise<void> {
+		let cellWithOutputs = notebook.document.cells[0].contents && notebook.document.cells[0].contents.outputs && notebook.document.cells[0].contents.outputs.length > 0;
+		assert(cellWithOutputs === true, 'First notebook cell has no outputs');
```

Comment:
Minor: usually expect messages to be in form "expected cell to have outputs", not "this has no outputs"

Reply:
Thanks